CHANGELOG.md

```diff
@@ -6,9 +6,33 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
 
 ## [??? (unreleased)]
 
+## [2019.3] - 2019-03-24
+
+### Added
+
+- New tool aktualizr-lite for anonymous TUF-only updates: [PR](https://github.com/advancedtelematic/aktualizr/pull/1107)
+- [Abort() API call](src/libaktualizr/primary/aktualizr.h): [PR](https://github.com/advancedtelematic/aktualizr/pull/1122)
+- [Option to print delegation metadata with aktualizr-info](src/aktualizr_info/main.cc): [PR](https://github.com/advancedtelematic/aktualizr/pull/1138)
+- Support for custom URIs for downloading targets: [PR](https://github.com/advancedtelematic/aktualizr/pull/1147)
+- [SendManifest() API call](src/libaktualizr/primary/aktualizr.h): [PR](https://github.com/advancedtelematic/aktualizr/pull/1176)
+
 ### Changed
 
-- Device installation failure result codes are deduced as concatenation of ECU failure result codes
+- Device installation failure result codes are deduced as concatenation of ECU failure result codes: [PR](https://github.com/advancedtelematic/aktualizr/pull/1145)
+- No longer require hashes and sizes of Tagets objects in Snapshot metadata: [PR](https://github.com/advancedtelematic/aktualizr/pull/1162)
+- [Updated documentation](README.adoc): [PR](https://github.com/advancedtelematic/aktualizr/pull/1164)
+
+### Removed
+
+- example.com is no longer set as the default URL when using garage-sign via garage-deploy: [PR](https://github.com/advancedtelematic/aktualizr/pull/1169)
+- OPC-UA secondary support: [PR](https://github.com/advancedtelematic/aktualizr/pull/1177)
+
+### Fixed
+
+- Check for updates even if sending the manifest fails: [PR](https://github.com/advancedtelematic/aktualizr/pull/1186)
+- Correctly handle empty targets metadata: [PR](https://github.com/advancedtelematic/aktualizr/pull/1186)
+- Various OSTree-related memory leaks: [PR #1114](https://github.com/advancedtelematic/aktualizr/pull/1114), [PR #1120](https://github.com/advancedtelematic/aktualizr/pull/1120), and [PR #1179](https://github.com/advancedtelematic/aktualizr/pull/1179)
```

Comment:
minor: #1179 is technically not a leak fix but merely an update of the suppression list (for supposed false positives) for the test suite.

Reply:
True. I can update it once we we figure out #1192.

Reply:
Done and now ready for review.